.github/workflows/main.yml

```
@@ -7,6 +7,13 @@ on:
 jobs:
   tests:
     name: tests-python${{ matrix.python-version }}-${{ matrix.os }}
+    rules:
+      - if: ${{ matrix.os }} == "windows-latest"
+        variables:
+          run_tests_command: "tox -e py"
+      - if: ${{ matrix.os }} != "windows-latest"
+        variables:
+          run_tests_command: "tox -e jax"
```

I don't think `rules` is a valid key. Reading the env variable section of github actions's documentation I'm not sure there is a way to do conditional variables like this: https://docs.github.com/en/actions/learn-github-actions/environment-variables

There might be a way with inline expressions but that'll be a bit harder to read imo if possible. I think my earlier suggestion is the best path forward here. Where we just do an if else in the steps to run either job based on the runner

```suggestion
    env:
      run_tests_command: ${{ 
        variables:
          run_tests_command: "tox -e py"
      - if: ${{ matrix.os }} != "windows-latest"
        variables:
          run_tests_command: "tox -e jax"
```

---

Thanks, I had tried that but I think it wasn't quite right, and then got lost in some random pages through google :P